resources/puppetlabs/lein-ezbake/template/global/ext/fpm.rb

```diff
@@ -181,25 +182,28 @@
     options.systemd_el = 1
   elsif options.operating_system == :el && options.os_version >= 7 # systemd el
     if ! options.is_pe
+      # https://bugzilla.redhat.com/show_bug.cgi?id=2224427
```

Comment:
This bug is resolved, so it could be dropped. But this at least provides a reference of why it's there.